src/System.Private.Interpreter/src/Internal/Runtime/Interpreter/ILInterpreter.cs

```diff
@@ -2449,10 +2450,10 @@ private void InterpretLoadElement(int token)
         // Holds information about a method call
         private struct MethodCallInfo
         {
-            public int ArgCount;
+            public TypeDesc OwningType;
+            public bool IsStatic;
```

Comment:
Is this always a cached value of `Signature.IsStatic` ? 

If yes, it is really worth caching it?

Reply:
Good point. There's no need to cache it since I'm the signature type is already set, I can just get this property from there